src/main/java/net/greghaines/jesque/worker/WorkerImpl.java

```diff
@@ -507,12 +507,14 @@ protected String getNextQueue() throws InterruptedException {
      */
     protected String pop(final String curQueue) {
```

Comment:
Don't we wanna add unit test to test\validate the arguments that are being passed in each case ?